docs/api-reference/next/image.md

```diff
@@ -39,16 +40,15 @@ We can serve an optimized image like so:
 
 ```jsx
 import Image from 'next/image'
+import profilePic from '../me.png'
 
 function Home() {
   return (
     <>
       <h1>My Homepage</h1>
       <Image
-        src="/me.png"
+        src={profilePic}
```

Comment:
Should we show code examples of:

- Static Image
- Dynamic src (fetching from a CMS)
  - Manually adding blurDataURL example?

Reply:
Perhaps, it could get busy.

We also need to update these pages:

https://nextjs.org/docs/basic-features/image-optimization
https://nextjs.org/docs/basic-features/static-file-serving